arm/proofs/aes.ml

```diff
@@ -0,0 +1,393 @@
+(*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0 OR ISC OR MIT-0
+ *)
+
+needs "Library/words.ml";;
+needs "compute.ml";;
+
+(** TODO:define a recursive definition for word_join_list and its CONV *)
+
+let pp_print_num fmt tm =
+  let n = dest_numeral tm in
+  pp_print_string fmt (string_of_num_hex n) in
+install_user_printer("pp_print_num",pp_print_num);;
```

Comment:
Are all numerals printed as hex after this point, even for proofs not using aes?

Reply:
Sorry, this PR is not ready yet. I'll turn it from draft to official once it's done.

Reply:
oops, sorry. okay!

Reply:
NP. Sorry for the confusion.

Reply:
Moved to a new file. Now it will only be used when installed.